src/main/java/uk/gov/laa/ccms/caab/controller/application/summary/OpponentsSectionController.java

```diff
@@ -275,4 +554,49 @@ private void populateConfirmSharedOrganisationDropdowns(final Model model) {
     model.addAttribute("relationshipsToClient", combinedLookup.getT2().getContent());
   }
 
+  private void populateOrganisationCreateDropdowns(final Model model) {
```

Comment:
if common lookups can we use the dropdownBuilder?

Reply:
done